// hataya hun

Manasrai2000 opened the pull request.

Manasrai2000 committed:
// hataya hun

Manasrai2000 committed:
sahi kiya hun

Manasrai2000 committed:
color change kar diya hun

VidyarthiAdda commented on index.html:
```diff
@@ -73,7 +73,7 @@
 .tblNoBorder td   { border: 0; }
 
 
-// -->
```
why this comment

VidyarthiAdda commented on thememxpro.com/html/hactora/faq.html:
```diff
@@ -2,443 +2,494 @@
-                        </ul>
+                                    <div class="acc-content">
+                                        <div class="content">
+                                            <div class="text">Deciding the right career for yourself can be tricky. Career Central helps you choose a suitable stream right after XII - Science/Commerce/Humanities or Arts - as per your aptitude and liking. It opens up a world of career options before you, where you can read about a career, its job profile, career opportunities, eligibility for the  ...
```
why this change?

VidyarthiAdda commented on thememxpro.com/html/hactora/index.html:
```diff
@@ -198,10 +198,10 @@
                                 <div class="text">Manufacturing Technology Supports Factories around the World. <br>Factories are at the heart of manufacturing.</div>
                                 <div class="link-box">
-                                    <a href="about.html" class="theme-btn btn-style-one"><span>About Us</span></a>
+                                    <a href="about.html" class="theme-btn btn-style-one" style="z-index: 11; position: absolute;"><span>About Us</span></a>
```
can we keep it default?

Manasrai2000 committed:
Add files via upload

Manasrai2000 committed:
card added

ha bhai 2 sections hai or usake saath images ki folder me usake lea images hai

Manasrai2000 committed:
images updated as per cards

live classes ka daal diya hun or educators ka images or url update kar diya hun
